include/libnfs-private.h

```diff
@@ -205,6 +205,8 @@ struct tls_context {
 };
 #endif /* HAVE_TLS */
 
+#define INC_STATS(rpc, stat) ++((rpc)->stats.stat)
```

Comment:
Should this use atomic_fetch_add_explicit() ?
Right now I think all callpaths are guaranteed to be from the same thread  but future updates and additional stats might be added from different threads and this could become racy, unless the multithreading mutex is held.

Reply:
Let's not add atomic till we need it. Since these are RPC stats the updates to them would be done by a single thread and in other places it'll be already protected by the rpc mutex.

Reply:
Also added a note to call this out in case we add more counters in future.

Reply:
Also, the counters are updated correctly only for clients. I see some code to allow libnfs based servers, but I'm not sure how complete is that.

Reply:
We can fix that later.
There are a couple of severs built on libnfs I am aware of but they are all for internal dedicated use in some processing workflows and not general purpose of public.